### PR TITLE
Jcs/core

### PR DESCRIPTION
**Pull request by jSylvestre**

### CODE REVIEW

File: Hippo.Web/Hippo.Web.csproj [hunk 1]
```diff
@@ -14,7 +14,16 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.Authentication.OpenIdConnect" Version="6.0.1" />
-    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="6.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.1">
+      <PrivateAssets>all</PrivateAssets>
```
Comment (jSylvestre) [inline Hippo.Web/Hippo.Web.csproj:20]: @srkirkland or @sprucely 
Know what this is?
Should I remove this section?

Comment (sprucely) [reply Hippo.Web/Hippo.Web.csproj:20]: Not certain, but I think that's to support some of Visual Studio's schema/query designer tools.

Comment (srkirkland) [reply Hippo.Web/Hippo.Web.csproj:20]: I think you need it to be able to run the migration scripts, like dotnet ef add migration or whatever the syntax is

File: Hippo.Web/Hippo.Web.csproj [hunk 2]
```diff
@@ -14,7 +14,16 @@
...
+    <PackageReference Include="Microsoft.AspNetCore.SpaServices.Extensions" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="6.0.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite.NetTopologySuite" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="6.0.1" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer.NetTopologySuite" Version="6.0.1" />
```
Comment (srkirkland) [inline Hippo.Web/Hippo.Web.csproj:26]: dump these -- it's only needed for Geo stuff and we aren't doing that in this proj

**Pull request merged**